src/nimblepkg/nimscriptapi.nim

```diff
@@ -178,10 +184,12 @@ template task*(name: untyped; description: string; body: untyped): untyped =
   proc `name Task`*() = body
 
   nimbleTasks.add astToStr(name)
+  if astToStr(name).len > nimbleTasksMaxNameLen:
+    nimbleTasksMaxNameLen = astToStr(name).len
```

Comment:
Nit: you can do this in `onExit` and save the need for another global variable

Reply:
Done. I wanted to avoid an extra loop, but normally there are few tasks, so not critical.